pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -182,16 +182,16 @@ in {
     # incompatibleKernelVersion = "4.19";
 
     # this package should point to a version / git revision compatible with the latest kernel release
-    version = "0.8.0-rc3";
+    version = "2019-03-29";
+    rev = "1b939560be5c51deecf875af9dada9d094633bf7";
 
-    sha256 = "0wmkis0q2gbj7sgx3ipxngbgzjcf7ay353v3mglf2ay50q4da5i7";
+    sha256 = "18isg2i8ig6rp1r4ylhz1n5shksrhkzf0mpadhai71v0y21599n0";
     isUnstable = true;
 
     extraPatches = [
-      # in case this gets out of date, just send Mic92 a pull request!
```

Comment:
Please do that instead of deleting the comment.

Reply:
On a related note, would you be interested in upstreaming this a version of this patch? We have an opportunity to add NixOS to the ZFSoL CI / build farm, and I think not requiring patches would be a step in that direction.

Reply:
> Please do that instead of deleting the comment.

This just got deleted by accident.

Reply:
@grahamc sounds good to me. Do you have some contacts in that direction?